src/Commands/CreateCommand.php

```diff
@@ -172,12 +176,18 @@ public function create($options = ['env' => null, 'element' => null, 'skip' => n
                                     }
                                 }
                                 if ($backup) {
-                                    $env->getBackups()->create($options)->wait();
-                                    $message = 'Created a backup of the {element} for {site_env}.';
+                                    if ($async) {
+                                      $env->getBackups()->create($options);
+                                      $message = 'Creating a backup of the {element} for {site_env}.';
+                                    }else{
+                                      $env->getBackups()->create($options)->wait();
+                                      $message = 'Created a backup of the {element} for {site_env}.';
+                                    }
```

Comment:
Since the $message line is the same, you could put it below the conditional:
```
if ($async) {
  $env->getBackups()->create($options);
} else {
  $env->getBackups()->create($options)->wait();
}
$message = 'Created a backup of the {element} for {site_env}.';
```